modules/dmrpp_module/DmrppCommon.cc

```diff
@@ -140,12 +140,15 @@ void DmrppCommon::parse_chunk_dimension_sizes(const string &chunk_dims_string)
  * @brief Parses the text content of the XML element h4:chunkDimensionSizes
  * into the internal vector<unsigned int> representation.
  *
- * @param compression_type_string One of "deflate" or "shuffle."
+ * @param compression_type_string
  */
 void DmrppCommon::ingest_compression_type(const string &compression_type_string)
 {
     if (compression_type_string.empty()) return;
+    d_filters = compression_type_string;
 
+#if 0
+    // old code utilizing booleans, SBL 10.26.21
```

Comment:
And remove this. 

You can remove all the commented out and #if 0 ... #endif code at this point and commit that using by including [skip ci] in the commit message. That will us from running the whole CI/CD process for changes that don't affect the code at all.